### ISSUE TITLE
winget install anything results in: "0x80004002 : No such interface supported"

### ISSUE THREAD

**Issue by theinventor** [microsoft/winget-cli#1109]
# Brief description of your issue
I'm running cmd as administrator and trying to get anything at all to install, I've tried random packages like cpu-z, evernote, both are giving this error "0x80004002 : No such interface supported"

(I did search google and your issues here, I can't find this error string anywhere)

# Steps to reproduce

Start, cmd, run as administrator, "winget install cpu-z"

# Expected behavior

it installs?

# Actual behavior

```
D:\>winget install cpu-z
Found CPUID CPU-Z [CPUID.CPU-Z]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.cpuid.com/cpu-z/cpu-z_1.96-en.exe
An unexpected error occurred while executing the command:
0x80004002 : No such interface supported
```

# Environment

```none
D:\>winget --info
Windows Package Manager v1.0.11451
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.17134.1845
Package: Microsoft.DesktopAppInstaller v1.11.11451.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir


Any other software?
```

**Comment by jedieaston** [comment]
I'm guessing no, but does it work if you run it from a non-admin command prompt and let it UAC the installer itself?

**Comment by theinventor** [comment]
I get the same thing for any install I try, non-admin or admin - thanks!

**Comment by denelon** [comment]
@theinventor can you submit [Feedback](https://aka.ms/winget-feedback) and share the URL?

**Comment by theinventor** [comment]
> @theinventor can you submit [Feedback](https://aka.ms/winget-feedback) and share the URL?

Ok, I submitted and put this issue URL on it. Hope that's what you meant :)

**Comment by denelon** [comment]
@theinventor

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).

Can you confirm if this is still happening?
